integration/e2e/metrics_promql.go

```diff
@@ -0,0 +1,283 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	dto "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/expfmt"
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/prometheus/prometheus/promql"
+	"github.com/prometheus/prometheus/storage"
```

Comment:
Have you measured if there's any significative negative impact in integration tests compilation time due to these imports?

Reply:
I expect that compilation time is small compared to actually running these tests, but I haven't timed it.

Reply:
We import those anyway.